governance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Governance 1.0 DRAFT WIP
+description: Governance 1.0 discussion draft
+---
+# Governance 1.0 `DRAFT WIP`
+
+This document contains a work in progress (WIP) draft of the initial governance of the Op1st project (Governance 1.0).
+
+It also has a roadmap for additions coming in the next iteration of governance (Governane 1.1).
+
+## Op1st Community Governance 1.0 (MVG)
+
+A healthy open source project is one that demonstrates open practices, uses open infrastructure, and cultivates an open culture with the goal of becoming more sustainable.
+
+### Principles
+As a new cloud-native community growing up at the footsteps of massively scaled and successful projects such as Kubernetes and OpenInfra, we want to be careful to learn from those who came before us, while forging our own path into the challenges we've chosen to undertake.
+
+To stand up this Governance for a 1.0 good-enough minimal-viable-governance (MVG), we are looking to our key predecessors for some of our starting points.
+As an initial set of principles we are combining elements of the [Kubernetes community principles](https://github.com/kubernetes/community/blob/master/governance.md#principles) and the [OpenInfra Foundations Four Opens](https://openinfra.dev/four-opens/).
+
+#### Op1st Community Principles 1.0
+Our mission requires us to use open source principles in ways beyond licensing the code we write, but at the core we are an open source community and want to adhere to the best possible principles for creating an open, welcoming, and inclusive community.
+
+These are the Op1st community principles we adhere to:
+1. **Open Source** : Any content we create is released under an open source license. Content includes software, documentations, configuration files, and training materials.
+  - This is crucial to fulfilling our mission around open sourcing operations
+2. **Welcoming and respectful** : We intend to be an open, welcoming, and inclusive community.
+Diversity is our strength.
+We have and follow a Code of Conduct.
+  - We currently follow the OpenInfra Foundation Code of Conduct as a member of the OpenInfra Labs.
+3. **Open Design** : We follow a transparent and open process for planning and designing any content, services, and features of Op1st.
+Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles.
+4. **Open Work** : Two of our core tools for successful community building are transparency and accessibility.
+To support that, work and collaboration must be done in public to the greatest degree possible.
+Refer to the section on Community Groups and SIGs in particular for details on how this is performed.
```

Comment:
You can link to the sections: 
https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#section-links

Reply:
Thanks, I think this is the correct syntax (the links are accurate, at least currently in the MD file in my fork rendered by GitHub ...)

```suggestion
Refer to the section on [Community Groups](#community-groups) and [SIGs](#sigs) in particular for details on how this is performed.
```